bin/i18n/hourofcode_crowdin.yml

```
@@ -19,24 +19,12 @@ files: [
     "translation": "/%language%/hourofcode/%original_file_name%",
     "update_option": "update_as_unapproved"
   }, {
-    "source": "/how-to/*.md",
-    "translation": "/%language%/hourofcode/how-to/%original_file_name%",
+    "source": "/**/*.md",
+    "translation": "/%language%/hourofcode/**/%original_file_name%",
     "update_option": "update_as_unapproved"
   }, {
-    "source": "/promote/*.md",
-    "translation": "/%language%/hourofcode/promote/%original_file_name%",
+    "source": "**/*.md.partial",
+    "translation": "/%language%/hourofcode/**/%original_file_name%",
```

I believe this should be
```suggestion
    "translation": "/%language%/hourofcode/**/%file_name%.md",
```

---

Fixed.